src/core/Table/TableRowMemoized.tsx

```diff
@@ -17,14 +18,18 @@ import { CSSTransition } from 'react-transition-group';
  */
 const TableRow = <T extends Record<string, unknown>>(props: {
   row: Row<T>;
-  rowProps: TableRowProps;
+  rowProps?: (
+    rowData: T,
+  ) => React.HTMLAttributes<HTMLDivElement> &
+    React.RefAttributes<HTMLDivElement>;
```

Comment:
Update types to `React.ComponentPropsWithRef<'div'>` to match

Reply:
Fixed.